my-app/src/app/home/quick-links/quick-links.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component} from '@angular/core';
+
+@Component({
+  selector: 'quick-links',
+  templateUrl: './quick-links.component.html',
+  styleUrls: ['./quick-links.component.css']
+})
+export class QuickLinksComponent  {
+
+  
+
+  ngOnInit() {
```

Comment:
This is an ngOnInit being used in the feature-home-page branch, along with many others.